Api/DeliveryStatus.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ProductBundle\Api;
+
+use Sulu\Bundle\ProductBundle\Entity\DeliveryStatus as Entity;
+use JMS\Serializer\Annotation\VirtualProperty;
+use JMS\Serializer\Annotation\SerializedName;
+use JMS\Serializer\Annotation\ExclusionPolicy;
+use Sulu\Component\Rest\ApiWrapper;
+use Sulu\Bundle\ProductBundle\Entity\DeliveryStatusTranslation;
+
+/**
+ * The DeliveryStatus class which will be exported to the API
+ *
+ * @package Sulu\Bundle\ProductBundle\Api
```

Comment:
I think we want to exclude these annotations (`@pacakge`) as they are just repeating information (i.e. the Namespace). /cc @chirimoya @danrot